wremnants/combine_theory_helper.py

```diff
@@ -45,20 +46,28 @@ def configure(self, resumUnc, np_model,
         self.set_resum_unc_type(resumUnc)
         self.set_np_model(np_model)
         self.set_propagate_to_fakes(propagate_to_fakes)
+        self.set_minnlo_unc(minnlo_unc)
 
         self.tnp_magnitude = tnp_magnitude
         self.mirror_tnp = mirror_tnp
         self.pdf_from_corr = pdf_from_corr
         self.pdf_action = pdf_action
         self.scale_pdf_unc = scale_pdf_unc
-        self.minnloScaleUnc = minnloScaleUnc
+        self.minnlo_unc = minnloScaleUnc
```

Comment:
Use either the "set_minnlo_unc" function or explicit set the minnlo_unc, but now both